mypy/server/astmerge.py

```diff
@@ -160,7 +160,7 @@ def replacement_map_from_symbol_table(
         ):
             new_node = new[name]
             if (
-                type(new_node.node) == type(node.node)  # noqa: E721
+                type(new_node.node) == type(node.node)
```

Comment:
Should this use `is` instead of `==`?

Reply:
We could although I'm uncertain how mypyc optimizes that and thus what the performance impact would be. That's why I chose to add `E721` to the ignore list. _There are three more instances which ruff would have suggested to rewrite._